front/src/pages/Home/Home.tsx

```diff
@@ -1,48 +1,36 @@
 import React from 'react';
-import { useQuery } from '@tanstack/react-query';
-import { todosAPI } from '../../api/todos-api';
-import { TodosArr } from '../../types';
-import { useDispatch } from 'react-redux';
-import { AxiosError } from 'axios';
-import { addMessage } from '../../redux/messages-reducer';
 import styles from './Home.module.scss';
 import Loader from '../../components/Loader';
+import useUpdateTodo from '../../hooks/useUpdateTodo';
+import useFetchTodos from '../../hooks/useFetchTodos';
 
 
-const Home = (): JSX.Element => {
-  const dispatch = useDispatch();
+const Home = () => {
 
   const {
     data: todos = [],
     isFetching,
-  } = useQuery<TodosArr, AxiosError>(
-    ['todos'],
-    todosAPI.getAllTodos,
-    {
-      onError: (error) => {
-        if (error?.message) {
-          dispatch(addMessage({
-            text: error.message,
-          }));
-        }
-      },
-    }
-  );
+  } = useFetchTodos();
+  const updateTodoMutation = useUpdateTodo();
```

Comment:
this is pretty minor, but with destructuring it would be even better(while it's already good enough):

```
const { mutate: updateToDo } = useUpdateTodo();
...
const handleTodoClick = (id: string, isDone: boolean) => {
  updateTodo({ id, isDone });
}
```

Reply:
I agree. Done.